runtime/kusama/src/lib.rs

```diff
@@ -1082,22 +1082,22 @@ impl InstanceFilter<Call> for ProxyType {
 			),
 			ProxyType::Governance => matches!(
 				c,
-				Call::Democracy(..) |
-					Call::Council(..) | Call::TechnicalCommittee(..) |
-					Call::PhragmenElection(..) |
-					Call::Treasury(..) | Call::Bounties(..) |
-					Call::Tips(..) | Call::Utility(..)
+				Call::Democracy(..)
+					| Call::Council(..) | Call::TechnicalCommittee(..)
```

Comment:
errrr I have no idea why my vscode is doing this. 

When I use the default formatter in the editor, it does this. It seems like it is using a different version of `cargo fmt` in there. When I do `cargo +nightly fmt` in the root, it is all good. Anyways, will revert later.